Project-Euler/Project-Euler/Problems.cs

```diff
@@ -150,5 +150,36 @@ private static bool IsPalendromic(string characters)
 
             return true;
         }
+
+        public static void Problem5()
+        {
+            Console.WriteLine("\n\t5)What is the smallest positive number that is evenly divisible by all of the numbers from 1 to 20?");
+
+            int evenlyDivisibleNumber = 0;
+
+            for(int i = 20; i < int.MaxValue; i += 20)
+            {
+                if(IsDivisibleFrom1To20(i))
+                {
+                    evenlyDivisibleNumber = i;
+                    break;
+                }
+            }
+
+            Console.WriteLine($"\n\tSolution: {evenlyDivisibleNumber}");
+        }
+
+        private static bool IsDivisibleFrom1To20(int number)
+        {
+            for(int i = 1; i < 21; i++)
```

Comment:
- You don't have to check for 1, because every int is divisible by 1
- You don't have to check for 2, 4, or 8, because we check for 16
- You don't have to check for 3, 6, or 9, because we check for 18
- You don't have to check for 5 or 10, because we check for 20
- You don't have to check for 7, because we check for 14